numpy/core/fromnumeric.py

```diff
@@ -3148,8 +3148,8 @@ def size(a, axis=None):
     See Also
     --------
     shape : dimensions of array
```

Comment:
should be numpy.ma.shape.

Reply:
> should be numpy.ma.shape.

This is the docstring for `np.size`, so it should be referencing functions from the `numpy` namespace, not `np.ma` I think.